types/culori/src/common.d.ts

```diff
@@ -57,10 +57,17 @@ export type Color =
     | Xyz65
     | Yiq;
 
+export type Gamut =
+    | P3
+    | Rec2020
+    | Rgb;
+
 export type NonEmptyArray<T> = [T, ...T[]];
 
 export type Mode = Color["mode"];
 
+export type RgbGamut = Gamut["mode"];
```

Comment:
```suggestion
export type GamutMode = Gamut["mode"];
```

Reply:
@bijela-gora Changes should be present in the latest push!